mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/StoredRecordChunksKafkaHandler.java

```diff
@@ -111,10 +109,6 @@ public StoredRecordChunksKafkaHandler(@Autowired @Qualifier("recordsPublishingSe
             LOGGER.error("RecordsBatchResponse processing has failed with errors correlationId: {} chunkNumber: {}", correlationId, chunkNumber, th);
             return Future.failedFuture(th);
           });
-      } catch (IOException e) {
```

Comment:
probably it makes sense not to delete try ... catch..., we can change IOException to Exception because we can't catch exceptions from Json.decodeValue (as example)

Reply:
Thanks Alex, yes it is discussed with Ruslan and it is done.